service-registry/src/main/resources/application-dev.yml

```
@@ -21,8 +35,8 @@
 logging:
   level:
     root: info
-    com.netflix.discovery: debug # Eureka 클라이언트 디버깅 (등록/갱신 확인)
-    com.netflix.eureka: debug # Eureka 서버 디버깅 (인스턴스 상태 추적)
-    org.springframework.cloud.netflix.eureka: debug # Spring Cloud Eureka 설정 디버깅
-    # 필요할 경우만 trace (예: 등록 누락 등 이슈 추적 시 사용)
-    # com.netflix.discovery: trace
+    com.netflix.discovery: info # Eureka 클라이언트 로깅
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)